src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/DoNotPrefixEnumValuesWithTypeName.cs

```diff
@@ -50,14 +51,19 @@ private static void AnalyzeNamedType(SymbolAnalysisContext context)
         {
             var symbol = (INamedTypeSymbol)context.Symbol;
             IEnumerable<ISymbol> enumValues;
-            if (symbol.TypeKind != TypeKind.Enum || !(enumValues = symbol.GetMembers().Where(m => m.Kind == SymbolKind.Field)).Any())
+            if (symbol.TypeKind != TypeKind.Enum || !(enumValues = symbol.GetMembers().Where(m => m.Kind == SymbolKind.Field && !m.IsImplicitlyDeclared)).Any())
```

Comment:
Excluding implicitly declared enums. This impacts the behavior of the current rule so if needed I could keep them for the 75% case.